terraform/modules/external_domain_broker/resources.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_route53_zone" "zone" {
-  name    = "${var.hosted_zone}"
+  name    = "external-domains-${var.stack_description}.cloud.gov"
```

Comment:
Noting that while pairing during a walkthrough review of this, we noticed the name was wrong and needed to be fixed to start with `external-domains` (the `s` was missing).

It's already fixed here as a part of the review.